scripts/xtensa-build-all.sh

```diff
@@ -329,6 +329,12 @@ do
 			HOST="xtensa-rn-elf"
 			XTENSA_TOOLS_VERSION="RF-2016.4-linux"
 			;;
+		mt8195)
+			PLATFORM="mt8195"
+			XTENSA_CORE="hifi4_8195_PROD"
+			HOST="xtensa-mt8195-elf"
+			XTENSA_TOOLS_VERSION="RI-2019.1-linux"
+			;;
 
```

Comment:
Modifying xtensa-build-all.sh should be a separate patch with a commit message like this

```
scripts: Add mt8195 to build script
```